packages/integration-tests/test/actions.spec.ts

```diff
@@ -65,27 +70,118 @@ const deposit = async (amount, value) => {
   const receipt = await watcher.getL2TransactionReceipt(msgHash)
 }
 
-describe('Messages', async () => {
+const withdraw = async (value) => {
+  const L2Messenger = new Contract(l2MessengerAddress, l2MessengerJSON.interface, l2Wallet)
+  const calldata = simpleStorage.interface.encodeFunctionData('setValue', [value])
+  const l2ToL1Tx = await L2Messenger.sendMessage(
+    simpleStorage.address,
+    calldata,
+    5000000,
+    { gasLimit: 7000000 }
+  )
+  await l2Provider.waitForTransaction(l2ToL1Tx.hash)
+  const count = (await simpleStorage.totalCount()).toString()
+  await sleep(60000)
```

Comment:
What is this sleep for?

Reply:
I chatted with @K-Ho and we're going to remove that and implement the withdraw function using the watcher instead. We discussed leaving it as is at the moment since this is disabled and then I'm going to open a new PR for finishing the withdrawal test in the next iteration.

Reply:
We should be careful about getting in the habit of adding dead code to the codebase because of the risk of something else coming up and the dead code never being removed. In this case its the `watcher` and `initWatcher` as well as the skipped withdrawal tests

Reply:
The watcher and initWatcher is actually used by the deposit test, but I'll remove the withdraw function and withdrawal test for now and just re-add them to the branch I'm going to open for withdrawals